sdk/keyvault/azure-security-keyvault-jca/README.md

```diff
@@ -190,6 +365,143 @@ Please replace `${KEY_VAULT}` with your key vault name and replace `${MANAGED_ID
 | PEM          | EC-HSM   | P-521                           | SHA512withECDSA | ✔       | 
 | PEM          | EC-HSM   | P-256K                          |                 | ✘       |
 
+## Using jarsigner with Azure Key Vault JCA
+The integration of Azure Key Vault JCA provider can be used with jarsigner to sign JAR files using certificates stored in Azure Key Vault. Below are the steps to configure and use jarsigner with this library.
+
+### Download and Configure JCA Provider Jar
+1. Download the latest [JCA](https://repo1.maven.org/maven2/com/azure/azure-security-keyvault-jca) Provider Jar.
+2. If you are using Java8, you need to add the JCA provider jar to the class path.
+    1. Place the jar under the folder `${JAVA_HOME}/jre/lib/ext`
+        - ![place-jar.jpg](resources/place-jar.png)
```

Comment:
This screenshot is outdated. It is better to change it to Windows style, because most of them are Windows style screenshots.